service.go

```diff
@@ -85,3 +88,40 @@ func (s *locationServiceImpl) initLocationsMap(terms []interface{}) {
 		s.locationLinks = append(s.locationLinks, locationLink{APIURL: s.baseURL + top.UUID})
 	}
 }
+
+func (s *locationServiceImpl) getLocationCount() int {
+	return len(s.locationLinks)
+}
+
+func (s *locationServiceImpl) getLocationIds() []string {
+	i := 0
+	keys := make([]string, len(s.locationsMap))
+
+	for k := range s.locationsMap {
+		keys[i] = k
+		i++
+	}
+	return keys
+}
+
+func (s *locationServiceImpl) reload() error {
```

Comment:
Same as in sections-tranformer: This is the duplication of init() method, could you please make it less redundant?